sip-examples/src/main/java/examples/busy/build.xml

```diff
@@ -42,13 +42,13 @@
         classpathref="project.class.path"
     />
      </target>
-    <target name="shootistlog" description= "Show the  log" >
-        <java classname="tools.tracesviewer.TracesViewer" fork="true">
-      <arg name="-server_file" value="shootistlog.txt" />
+        	<target name="shootistlog" description="Show the log">
+        	    <java classname="gov.nist.core.LogWriter" fork="true">
+        	        <arg line="-server_file shootistlog.txt" /> 
     </java>
      </target>
     <target name="shootmelog" description= "Show the  log" >
-        <java classname="tools.tracesviewer.TracesViewer" fork="true">
+        <java classname="gov.nist.core.LogWriter" fork="true">
```

Comment:
this should be removed since tracesviewer has been removed

Reply:
done